chapters/packages.tex

```diff
@@ -321,3 +321,146 @@ \section{External Resources}\label{external-resources}
 \filename{modelica://Modelica/Mechanics/C.jpg} is illegal.
 The references \filename{modelica://Modelica.Mechanics/C.jpg} and \filename{modelica://Modelica/C.jpg} must also refer to two distinct resources.
 \end{example}
+
+\section{Multilingual Descriptions}\label{multilingual-descriptions}
+
+\begin{nonnormative}
+Descriptive texts in a model or library are usually formulated in English.
+This section describes how a tool can present the library in another language.
+Translated Modelica text is provided by external files, so that no modification of the original Modelica text is required.
+\end{nonnormative}
+
+The texts in following Modelica constructs should be translated:
```

Comment:
Before we can work out the example starting with _Suppose the following expression appears in a context to be translated:_ (proposed in a comment), we need to normatively define the translatability of something which isn't a concatenation of string literals:
```
  annotation(…, Text(string = "Hello, " + getInstanceName()), …);
```
Is the string `"Hello, "` translatable in this case?

Reply:
I added a text below related to that, but I don't see that as blocking.

Reply:
Resolving now that we have another conversation on this topic.